libraries/core-react/src/Select/NativeSelect/Select/Select.tokens.ts

```diff
@@ -0,0 +1,69 @@
+import { tokens } from '@equinor/eds-tokens'
+import type { Typography, Spacing } from '@equinor/eds-tokens'
+
+const {
+  colors,
+  spacings: { comfortable },
+  typography,
+} = tokens
+
+const spacings = {
+  input: {
+    left: comfortable.small,
+    right: comfortable.small,
+    top: '6px',
+    bottom: '6px',
+  },
+}
+
+export type SelectVariantProps = {
+  background: string
+  typography: Typography
+  color: string
+  spacings: {
+    input: Spacing
+  }
+  default: {
+    icon: {
+      width: string
+    }
+    border: {
+      outline: { color: string; width: string }
+      bottom: { color: string; width: string }
+    }
+    focus: {
+      border: {
+        width: string
+        color: string
+      }
+    }
```

Comment:
I think we have a type for `Border` and `Focus` in tokens types. Maybe try using them? Small-nude to conform tokens 🙈

Reply:
Did copy and paste from Text field - it feel's like that component is lagging behind